src/main/java/cn/nukkit/level/light/LightUpdate.java

```diff
@@ -0,0 +1,170 @@
+package cn.nukkit.level.light;
+
+import cn.nukkit.block.Block;
+import cn.nukkit.level.ChunkManager;
+import cn.nukkit.level.Level;
+import cn.nukkit.math.BlockVector3;
+
+import java.util.ArrayDeque;
+import java.util.HashSet;
+import java.util.Queue;
+import java.util.Set;
+
+/**
+ * author: dktapps
```

Comment:
@Creeperface01 It's other language of programming. You can delete that. ;)
You are author now because you "created". ;)